src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,231 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0.dev7 (8/10/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* A python_distribution target. (#10565)
+  `PR #10565 <https://github.com/pantsbuild/pants/pull/10565>`_
+
+* Enable Python dependency inference by default (#10520)
+  `PR #10520 <https://github.com/pantsbuild/pants/pull/10520>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow using file addresses on the command line (#10573)
+  `PR #10573 <https://github.com/pantsbuild/pants/pull/10573>`_
+
+* Targets act as aliases for their files (#10511)
+  `PR #10511 <https://github.com/pantsbuild/pants/pull/10511>`_
```

Comment:
I think probably an API change. This pretty dramatically changes how Pants behaves.